packages/store/src/yjs/index.ts

```diff
@@ -22,4 +24,9 @@ export class BlockSuiteDoc<
     const map = super.getMap(key);
     return createYMapProxy(map, config);
   }
+
+  @debug('transact')
```

Comment:
What's the difference between `transact` and `CRUD`?

Reply:
Just a tag name. The reason is crdt functions always have a transact function. That would cause duplicate logs